src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -73,20 +82,36 @@ internal class OnboardingSupervisor(
 
     private fun retrieveAssetsFromRouter() {
         if (StatsigConfig.useSkip) {
-            retrieveSkipTransferChains()
+            CoroutineScope(Dispatchers.Unconfined).launch {
```

Comment:
we should be storing this coroutine scope somewhere (and re-using for other coroutine launches).

Reply:
also let's add `SupervisorJob` as another coroutine context element.